volumes/converter_test.go

```diff
@@ -0,0 +1,107 @@
+package volumes
+
+import (
+	"reflect"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestPaastaVolumesToKubernetesVolumes(t *testing.T) {
+	fakeVols := []Volume{
+		Volume{
+			HostPath:      "/tmp/rw",
+			ContainerPath: "/tmp/bar",
+			Mode:          "RW",
+		},
+		Volume{
+			HostPath:      "/tmp/ro",
+			ContainerPath: "/tmp/bar",
+			Mode:          "RO",
+		},
+	}
+	volumeMounts, volumes := paastaVolumesToKubernetesVolumes(fakeVols)
+	expectedMounts := []corev1.VolumeMount{
+		corev1.VolumeMount{
+			Name:      "tmp-rw",
+			MountPath: "/tmp/bar",
+			ReadOnly:  false,
+		},
+		corev1.VolumeMount{
+			Name:      "tmp-ro",
+			MountPath: "/tmp/bar",
+			ReadOnly:  true,
+		},
+	}
+
+	// volumeType := corev1.HostPathDirectory
+	expectedVolumes := []corev1.Volume{
+		{
+			Name: "tmp-rw",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/tmp/rw",
+					// Type: &volumeType,
+				},
+			},
+		},
+		{
+			Name: "tmp-ro",
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/tmp/ro",
+					// Type: &volumeType,
+				},
+			},
+		},
+	}
+
+	if !reflect.DeepEqual(volumeMounts, expectedMounts) {
```

Comment:
Looking at https://flaviocopes.com/golang-comparing-values/, it seems that we probably can use just `==` instead of `reflect.DeepEqual`.
Can you please check that while you're here?

Reply:
Turns out that slices don't support `Equals`